index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portflio</title>
+    <link rel="stylesheet" href="nina.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Festive&family=Poppins&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+    <div class="container">
+        <div class="header hoverable">
+            <h1>My Logo</h1>
+            <img src="/photos/Enabled.png" alt="Hamburger Icon">
+        </div>
```

Comment:
- [ ] Please replace this `<div>` tag with `<header>` tag for semantic purposes.
- [ ] kindly replace the `<h1>` tag with a `<a>` tag for semantic purposes because when you click on your site logo it supposes to take you to the home page.
- [ ] Kindly wrap the hamburger icon `<img>` tag with a `nav` tag because semantically your navigation button and links suppose to be in a `<nav>` tag.